### DISCUSSION TITLE
Actividad 0 - Santillan Bruno Alejo

### DISCUSSION THREAD

**Discussion by bsantillan** [frlp-utn-ingsoft/catedra#56] [General]
https://github.com/bsantillan/Ingenieria-y-Calidad-de-Software.git

**Comment by RodrigoJacznik** [comment]
Hola,
buenos commits, la app funciona bien.
Actividad aprobada

**Comment by bsantillan** [comment]
Buenas profesor, genial.

Muchas gracias.
Saludos.
